README.md

```diff
@@ -6,7 +6,7 @@ It follows the [Controller pattern](https://godoc.org/github.com/kubernetes-sigs
 
 Most users will be able to use the top-level OpenShift Config API, which has a [Network type](https://github.com/openshift/api/blob/master/config/v1/types_network.go#L26). The operator will automatically translate the `Network.config.openshift.io` object in to a `NetworkConfig.networkoperator.openshift.io`.
 
-When the controller has reconciled and all its dependent resources have converged, the cluster should have an installed SDN plugin and a working service network. In OpenShift, the Cluster Network Operator runs very early in the install process -- while the boostrap API server is still running.
+When the controller has reconciled and all its dependent resources have converged, the cluster should have an installed SDN or OVN plugin and a working service network. In OpenShift, the Cluster Network Operator runs very early in the install process -- while the boostrap API server is still running.
```

Comment:
Let's say "an installed network plugin".

In general, we want the documentation to be generic. We don't want to say "OpenShift SDN or ovn-kubernetes" all over the place, because if we do that, then when we add Kuryr support we're going to have to say "OpenShift SDN, ovn-kubernetes, or Kuryr" in a bunch of those places, etc, etc, etc.